docs/geedocs/docsrc/answer/geeServerConfigAndSecurity/configureGeeVirtualHostForLDAP.rst

```diff
@@ -84,9 +84,9 @@ Configure a GEE virtual host for LDAP authentication
                AuthType Basic
                AuthName "LDAP LOGIN"
                AuthBasicProvider ldap
-               AuthLDAPURL "ldap://server.name.local:389/cn=Users,dc=domain,dc=google,dc=com?uid"\
-               AuthLDAPBindDN CN=geserver,OU=Users,OU=Yourorg,DC=DcNAME,DC=local
-               AuthLDAPBindPassword localuserpassword
+               AuthLDAPURL "ldap://ldapserver.example.com:389/ou=Users,dc=example,dc=com?uid"
+               AuthLDAPBindDN "someuser@example.com"
+               AuthLDAPBindPassword "somepassword"
                Require valid-user
             </Location>
 
```

Comment:
Can't comment where it belongs, but in AuthLDAPURL below, should be where to look for users, delete at